<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerConfig</class>
 <widget class="QDialog" name="ServerConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/network-icon.png</normaloff>:/images/network-icon.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QPushButton" name="pclBtnCloseM">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>440</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="pclTabConfigM">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pclTabConfigNetworkM">
    <attribute name="title">
     <string>Network configuration</string>
    </attribute>
    <widget class="QGroupBox" name="pclGrpNetworkConfM">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>5</y>
       <width>451</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>CAN configuration</string>
     </property>
     <widget class="QComboBox" name="pclCbbNetNomBitrateM">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>28</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="pclLblNetNomBitrateM">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nominal Bit-rate</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="pclCkbNetEnableM">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable network</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="pclCkbNetErrorFramesM">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Error frames</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="pclCkbNetCanFdM">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CAN-FD support</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="pclCkbNetListenOnlyM">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Listen-only</string>
      </property>
     </widget>
     <widget class="QLabel" name="pclLblNetDatBitrateM">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Data Bit-rate</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="pclCbbNetDatBitrateM">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>58</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pclPbtNetDevSpecificConfigM">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>110</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Device Specific Config</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="pclGrpNetworkStatM">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>451</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>CAN statistics</string>
     </property>
     <widget class="QProgressBar" name="pclPgbStatLoadM">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Bus load in percent</string>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
     <widget class="QLabel" name="pclLblStatMsgM">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Messages / sec:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="pclLblStatErrM">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Error frames:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="pclLblStatCanM">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CAN frames:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="pclCntStatCanM">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="pclCntStatErrM">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="pclCntStatMsgM">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>20</width>
       <height>341</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QGroupBox" name="pclGrpNetworkStatusM">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>290</y>
       <width>451</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>CAN status</string>
     </property>
     <widget class="QLabel" name="pclTxtStatusBusM">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Error active</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QToolButton" name="pclBtnStatusResetM">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>20</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="toolTip">
       <string notr="true">Reset CAN interface</string>
      </property>
      <property name="statusTip">
       <string notr="true"/>
      </property>
      <property name="whatsThis">
       <string notr="true"/>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0px;</string>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="icon">
       <iconset resource="../server.qrc">
        <normaloff>:/images/mc_reset_256.png</normaloff>:/images/mc_reset_256.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="popupMode">
       <enum>QToolButton::DelayedPopup</enum>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonIconOnly</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="pclTabConfigServerM">
    <attribute name="title">
     <string>Server configuration</string>
    </attribute>
    <widget class="QLabel" name="pclLblSrvPortM">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CAN server port</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="pclEdtSrvPortM">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>230</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="pclChkServerRemoteAccessM">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>481</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Allow access to server from remote computer</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="pclChkChangeBitrateM">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>481</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Allow application to change bit-rate settings</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pclTabConfigInfoM">
    <attribute name="title">
     <string>Information</string>
    </attribute>
    <widget class="QLabel" name="pclLblInfoIconM">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>271</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../server.qrc">:/images/canpie_fd_logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="pclLblInfoVersionM">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">CANpie Server Version 0.82</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>611</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../server.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pclBtnCloseM</sender>
   <signal>clicked()</signal>
   <receiver>ServerConfig</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
